src/index.js

```diff
@@ -60,17 +65,21 @@ function matchSorter(items, value, options = {}) {
  * @param {Array} keys - the keys to get values from the item for the ranking
  * @param {String} value - the value to rank against
  * @param {Object} options - options to control the ranking
- * @return {{rank: Number, keyIndex: Number}} - the highest ranking
+ * @return {{rank: Number, keyIndex: Number, itemThreshold: Number}} - the highest ranking
```

Comment:
Should we call this `keyThreshold` instead?

Reply:
Yup. Agree. I’ll make the change soon.